test/TestProjects/MgmtListMethods/readme.md

```diff
@@ -11,4 +11,7 @@ input-file: $(this-folder)/mgmtListMethods.json
 namespace: MgmtListMethods
 mgmt-debug:
   show-request-path: true
+
+override-operation-name:
+  FakeParentWithAncestorWithNonResChWithLocs_ListBySubscription: GetFakeParentWithAncestorWithNonResChWithLocs
```

Comment:
Should you change the configured name as now the generator automatically names `FakeParentWithAncestorWithNonResChWithLocs_ListBySubscription` to `GetFakeParentWithAncestorWithNonResChWithLocs`?